tests/Functional/Bundle/Maker/MakeFactoryTest.php

```diff
@@ -6,13 +6,89 @@
 use Symfony\Bundle\MakerBundle\Exception\RuntimeCommandException;
 use Symfony\Component\Console\Tester\CommandTester;
 use Zenstruck\Foundry\Tests\Fixtures\Entity\Category;
+use Zenstruck\Foundry\Tests\Fixtures\Entity\Foo;
 use Zenstruck\Foundry\Tests\Fixtures\Entity\Tag;
 
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
 final class MakeFactoryTest extends MakerTestCase
 {
+    /**
+     * @test
+     */
+    public function can_create_factory_with_relation_defaults(): void
+    {
+        $tester = new CommandTester((new Application(self::bootKernel()))->find('make:factory'));
+
+        $this->assertFileDoesNotExist(self::tempFile('src/Factory/FooFactory.php'));
+
+        $tester->execute(['entity' => Foo::class]);
+
+        $this->assertFileExists(self::tempFile('src/Factory/FooFactory.php'));
+        $this->assertSame(<<<EOF
+<?php
+
+namespace App\\Factory;
+
+use Zenstruck\\Foundry\\Tests\\Fixtures\\Entity\\Foo;
+use Zenstruck\\Foundry\\ModelFactory;
+use Zenstruck\\Foundry\\Proxy;
+
+/**
+ * @extends ModelFactory<Foo>
+ *
+ * @method static Foo|Proxy createOne(array \$attributes = [])
+ * @method static Foo[]|Proxy[] createMany(int \$number, array|callable \$attributes = [])
+ * @method static Foo|Proxy find(object|array|mixed \$criteria)
+ * @method static Foo|Proxy findOrCreate(array \$attributes)
+ * @method static Foo|Proxy first(string \$sortedField = 'id')
+ * @method static Foo|Proxy last(string \$sortedField = 'id')
+ * @method static Foo|Proxy random(array \$attributes = [])
+ * @method static Foo|Proxy randomOrCreate(array \$attributes = [])
+ * @method static Foo[]|Proxy[] all()
+ * @method static Foo[]|Proxy[] findBy(array \$attributes)
+ * @method static Foo[]|Proxy[] randomSet(int \$number, array \$attributes = [])
+ * @method static Foo[]|Proxy[] randomRange(int \$min, int \$max, array \$attributes = [])
+ * @method Foo|Proxy create(array|callable \$attributes = [])
+ */
+final class FooFactory extends ModelFactory
+{
+    public function __construct()
+    {
+        parent::__construct();
+
+        // TODO inject services if required (https://symfony.com/bundles/ZenstruckFoundryBundle/current/index.html#factories-as-services)
+    }
+
+    protected function getDefaults(): array
+    {
+        return [
+            // TODO add your default values here (https://symfony.com/bundles/ZenstruckFoundryBundle/current/index.html#model-factories)
+            'oneToOne' => BarFactory::new(),
+            'manyToOne' => BarFactory::new(),
```

Comment:
I believe this will change to:

```suggestion
            'oneToOne' => Zenstruck\Foundry\Tests\Fixtures\Factories\CategoryFactory::new(),
            'manyToOne' => Zenstruck\Foundry\Tests\Fixtures\Factories\CategoryFactory::new(),
```